### PR TITLE
Add caching detail in doc

### PR DESCRIPTION
**Pull request by xster**
From hackathon feedback

### CODE REVIEW

File: packages/flutter/lib/src/widgets/image.dart [hunk 1]
```diff
@@ -128,6 +128,8 @@ class Image extends StatefulWidget {
   /// Creates a widget that displays an [ImageStream] obtained from the network.
   ///
   /// The [src], [scale], and [repeat] arguments must not be null.
+  ///
+  /// By default, all network images are cached regardless of HTTP headers.
```
Comment (yjbanov) [inline packages/flutter/lib/src/widgets/image.dart]: Remove "By default"? I don't think users can do anything about it right now.

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
lgtm

**Comment by devoncarew** [comment]
Thanks for the doc update! lgtm

I'm doing some investigation into adding logging to the framework and having this exposed to tooling. So you could toggle different named log streams. In this case, something like `network.http` which would show whey requests were made, the result status code, and failures, and the elapsed time and data received.

File: packages/flutter/lib/src/widgets/image.dart (hunk 1 above)
Comment (xster) [reply packages/flutter/lib/src/widgets/image.dart]: done

**Pull request merged**